docs/scripts/og-image-stream.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { ImageResponse } from '@vercel/og';
```

Comment:
The code doesn't compile because of a missing `"fs"`import here

Reply:
yup